python-client/giskard/core/model.py

```diff
@@ -289,15 +289,38 @@ def __init__(self,
         self.data_preprocessing_function = data_preprocessing_function
         self.model_postprocessing_function = model_postprocessing_function
 
+    def post_processing(self, raw_prediction):
```

Comment:
to make it less confusing for an end user which post processing function he has to override, I'd make this method private by adding `_` prefix